config/samples/pipeline_v1beta1_pipelinerun.yaml

```diff
@@ -14,16 +14,8 @@ status:
   taskRuns:
     - taskRef:
       name: test-wizzbang-task-run-sd8f8dfasdfasdfas
-      nextTasks:
-        - taskRef:
-            name: build-push-task-run-sd8f8dfasdfasdfas
-      prevTasks: []
     - taskRef:
       name: build-push-task-run-sd8f8dfasdfasdfas
-      nextTasks: []
-      prevTasks:
-        - taskRef:
-          name: test-wizzbang-task-run-sd8f8dfasdfasdfas
```

Comment:
i'd still like to see some representation in the PipelineRun of the order the Tasks were executed in, any thoughts on how to do that in the new design?

Reply:
The passedConstraint controls that. If there are no constraints that means all tasks can run in any order or in parallel. Do you mean you want to see the actual order that happened in runtime?

Reply:
> Do you mean you want to see the actual order that happened in runtime?

Yes, exactly - especially since the failure of a Task would halt the execution of the rest of the Tasks, so it would be useful to see what actually executed and in what order